bcipy/helpers/list_processing.py

```diff
@@ -0,0 +1,20 @@
+"""Utility functions for list processing."""
```

Comment:
Would `helpers/list.py` work? It might make the module generic enough to pull other operations on lists here.